### DISCUSSION TITLE
does port 80 need to be open?

### DISCUSSION THREAD

**Discussion by melbruki** [nextcloud/all-in-one#4795] [Questions] [answered]
I installed the helm chart without a reverse proxy, and it looks like it has a certificate installed using letsencrypt, how did it get this certificate I dont have port 80 open on my firewall ? I thought that  port 80 need to be open for it to get a certificate also,

**Comment by szaimen** [comment, marked as answer]
> does port 80 need to be open?

Hi, obviously it doesnt because we use caddy for getting the certificate and it is able to use the TLS-ALPN challenge via port 443: https://letsencrypt.org/de/docs/challenge-types/#tls-alpn-01

**Reply by melbruki** [reply]
I see it is set to expire in 3 months does it renew automatically?

**Reply by szaimen** [reply]
Yes
